client/pages/fetch-data.jsx

```diff
@@ -25,21 +41,81 @@ export default class FetchData extends React.Component {
     this.willFetch(id);
   }
 
-  handleTeamDetails(id) {
-
-  }
-
   componentDidMount() {
 
     axios.request('/api/week-games/:date').then(response => {
       const fixtures = response.data;
       this.setState({
         fixturesList: fixtures,
-        isLoading: false
+        isLoading: false,
+        awayOdds: randomOdds(1, 5, 2),
+        homeOdds: randomOdds(1, 4, 2)
       }
       );
     });
+    axios.get('/api/wager-input').then(response => {
+      const pastBets = response.data;
+      this.setState({
+        gamesBetOn: pastBets
+      });
+    });
+  }
+
+  addWagerInput(newWager) {
+    const { fixtureId } = newWager.fixtureId;
+    const newArray = this.state.gamesBetOn.slice();
+    newArray.push(fixtureId);
+    axios.post('/api/wager-input', { newWager });
+  }
```

Comment:
Is the UI supposed to update after this part executes? You mentioned that the screen needs to be refreshed, but can you explain to me what you're trying to update? I'll help you structure the code so that the refresh isn't needed.